src/org-manager/containers/pending-pbas/pending-pbas.component.spec.ts

```diff
@@ -92,32 +92,36 @@ describe('PendingPBAsComponent', () => {
       fixture.detectChanges();
     });
 
-    it('should show a table with a single row that says there are none pending', () => {
-      const tds = utils.getDebugElements('td');
-      expect(tds.length).toEqual(1);
-      utils.checkText(tds[0], 'There are no pending PBAs');
+    it('should show there are no new PBA requests.', () => {
+      component.loaded$ = of(true);
+      fixture.detectChanges();
+      const messageContent = fixture.debugElement.nativeElement.querySelector('.govuk-heading-l').parentElement.textContent;
+      expect(messageContent).toContain('There are no new PBA requests.');
     });
 
     it('should show an appropriate count of zero in the title', () => {
       utils.checkText(utils.getTitle(), 'New PBAs (0)');
+
```

Comment:
You added a space here, but not on the next test - any particular reason?

Reply:
no particular reason so removed